fms/__main__.py

```diff
@@ -1,120 +1,121 @@
+import argparse
+import sys
 
-import sys, optparse
-from utils import TypeEnforcer as te
 from happi import Client
-from happi.item import OphydItem
-from containers import FMSRaritanItem, FMSBeckhoffItem, FMSSRCItem
-from typing import List
+from happi.errors import SearchError
 
-fms_happi_database = "fms_test.json"
-def validate():
-    client = Client(path=fms_happi_database)
-    print(type(client.validate()))
+from .add_sensor import add_sensor
+from .check_topology import check_topology
+from .happi.containers import FMSRaritanItem
+
+fms_happi_database = "/cds/home/n/nrw/fms/db.json"
 
-def get_all_src_status():
-    get_src_controllers()
-def get_src_controllers(client: Client=None) -> List[str]:
+
+def delete_sensor(sensor_name, client=None):
     if client is None:
         client = Client(path=fms_happi_database)
-    ret = client.search(name='booch')
-    print(ret)
+    item = client.find_item(name=sensor_name)
+
+    if type(item) is FMSRaritanItem:
```

Comment:
Suggestion: It might be helpful to have some kind of output / indication if are passed a non-FMSRaritanItem.  Otherwise this function just completes silently

Reply:
Added more deletion feedback.